lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -2266,6 +2277,9 @@ export class StandardController implements IController {
             )(silentRequest, cacheLookupPolicy).catch(
                 (cacheError: AuthError) => {
                     if (cacheLookupPolicy === CacheLookupPolicy.AccessToken) {
+                        this.logger.verbose(
+                            "Failed to retrieve token from cache"
```

Comment:
Redundant, will be logged as part of the invoke flow

Reply:
Yes, added with the initial fix, removing.